sycl/include/CL/sycl/ONEAPI/sub_group.hpp

```diff
@@ -97,6 +97,13 @@ void store(multi_ptr<T, Space> dst, const vec<T, N> &x) {
 
 } // namespace detail
 
+namespace ONEAPI {
+struct sub_group;
+}
+namespace experimental {
+ONEAPI::sub_group this_sub_group();
```

Comment:
```suggestion
sycl::sub_group this_sub_group();
```

Reply:
This wouldn't work. `sycl::sub_group` is an alias to `ONEAPI::sub_group`, defined in `sycl/sub_group.hpp`, which in turn includes `sycl/ONEAPI/sub_group.hpp`. So, I'd either reference an undefined symbol or create a cyclic dependency.
I can add one more alias to `sycl/ONEAPI/sub_group.hpp`, if you'd like. It seems it doesn't lead to ODR violation or something.